openedx/features/idea/views.py

```diff
@@ -30,4 +33,6 @@ class IdeaDetailView(View):
     template_name = 'features/idea/idea_details.html'
 
     def get(self, request, *args, **kwargs):
-        return render_to_response(self.template_name, {})
+        idea = get_object_or_404(Idea, pk=kwargs['pk'])
+        context = {'idea': idea}
+        return render_to_response(self.template_name, context)
```

Comment:
Is page will only be visible to logged in user. If so, shouldn't this class have `login_required` annotation